src/api/constants/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './Messages'
```

Comment:
I think messages could be moved back out of api, what do you think?
Since each command will have its own messages maybe we shouldn't share this